bin/reth/src/dirs.rs

```diff
@@ -293,6 +293,13 @@ impl<D> ChainPath<D> {
         self.0.join("blobstore").into()
     }
 
+    /// Returns the path to the local transactions backup file
+    ///
+    /// `<DIR>/<CHAIN_ID>/txpool-transactions-backup.rlp`
+    pub fn txpool_path(&self) -> PathBuf {
```

Comment:
this name should include `transactions`,

perhaps, `txpool_transactions_path`?